.github/workflows/ci.yaml

```diff
@@ -69,11 +69,11 @@ jobs:
       fail-fast: false
       matrix:
         tox-environments:
-          - integration-charm
-          - integration-scaling
-          - integration-provider
+          #- integration-charm
+          #- integration-scaling
+          #- integration-provider
```

Comment:
I suppose we should definitely re-introduce them to make sure there are no regression due to the large refactoring/code base update of the PR